src/style.scss

```diff
@@ -0,0 +1,148 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:regular,500,700&display=swap");
+
+body {
+  font-family: "Roboto", sans-serif ;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  padding: 0 16px;
+  width: 200px;
+  height: 406px;
+  border: 1px solid rgba(243, 243, 243, 1);
+
+  &__icon {
+    display: flex;
+    justify-content: center;
+    margin-top: 32px;
+    margin-bottom: 40px;
+    img {
+      width: 160px;
+      height: 134px;
+    }
+  }
+
+  &__title {
+    display: flex;
+    flex-direction: column;
+    margin: 0 auto;
+    font-family: inherit;
+    color: rgba(6, 11, 53, 1);
+  }
+
+  &__text {
+    text-align: left;
+    max-width: 166px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 150%; /* 18/12 */
+    padding-bottom: 4px;
+  }
+
+  &__code {
+    text-align: left;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 140%; /* 14/10 */
+    color: rgba(97, 96, 112, 1);
+    padding-bottom: 16px;
+  }
+
+  &__link {
+    display: flex;
+    border-radius: 5px;
+
+    height: 40px;
+    background: rgba(0, 172, 220, 1);
+    font-weight: 700;
+    color: #fff;
+    text-decoration: none;
+    font-size: 14px;
+    line-height: 114.29%; /* 16/14 */
+    justify-content: center;
+    text-transform: uppercase;
+    align-items: center;
+    margin-top: 16px;
+    transition: background-color 0.3s ease 0s;
+    &:hover {
+      background-color: #fff;
+      border: 1px solid rgba(0, 172, 220, 1);
+      color: rgba(0, 172, 220, 1);
+      box-sizing: border-box;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  &__star {
+    background: url("../src/images/star.svg") center no-repeat;
+    width: 16px;
+    height: 16px;
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+}
+
+.stars-card {
```

Comment:
you don't need this

use `card__stars` if you need to add external styles (margin)